Example/Messaging/Tests/FIRMessagingTest.m

```diff
@@ -59,8 +72,19 @@ @implementation FIRMessagingTest
 
 - (void)setUp {
   [super setUp];
-  _messaging = [FIRMessaging messagingForTests];
```

Comment:
can you also get rid of messagingForTests for FCM ?

Reply:
We'll need to wait until the IID tests change, but that can be done. I'll make sure to remove all other instances in Messaging first, though!